desktop-widgets/preferences/preferences_units.cpp

```diff
@@ -51,14 +51,14 @@ void PreferencesUnits::syncSettings()
 	QString unitSystem[] = {"metric", "imperial", "personal"};
 	short unitValue = ui->metric->isChecked() ? METRIC : (ui->imperial->isChecked() ? IMPERIAL : PERSONALIZE);
```

Comment:
Turn into `int`, please.

Reply:
I did not touch that, it was a short before, but I agree it should be an int. Thus PR tries only to implement qPrefUnits, and will be followed by a PR cleaning the use of units. I have added your suggestion to that branch.